### DISCUSSION TITLE
Mapping jk (simultaneously pressed) to Escape?

### DISCUSSION THREAD

**Discussion by glyh** [kovidgoyal/kitty#4672] [Q&A] [answered]
I'm trying to mapping jk to escape, just like in vim.  Here's the behavior I expect: 
1. Any sequences of js and ks, if not pressed simultaneously, will be perfectly inserted with no problem.
2. If j and k are pressed simultaneously, this would be triggered.

which is bascially the behavior of [kana/vim-arpeggio](https://github.com/kana/vim-arpeggio), if one use vim/neovim
I've found that kitty have the multi-key shortcuts, like `map j>k set_font_size 20`, but this does not have the behavior I expect. 
Also, I did not find a way to send keys in mappable actions. 

So could you help me out? Thanks a lot!

**Comment by kovidgoyal** [comment, marked as answer]
This is not possible. The multikey shortcuts have no concept of
simultenous or not. Once you assign a leader key then that key has to be
used for a shortcut, it cannot be used for shortcut or typing.

You can possibly fake it by doing something like

map j>k whatever
map j>x send_text all jx
repeat for all possible j<whatever> combinations

**Reply by glyh** [reply]
I got it, thank you.
